.github/CIP-TEMPLATE.md

```diff
@@ -32,7 +32,10 @@ License: CC-BY-4.0
 <!-- A clear explanation that introduces the reason for a proposal, its use cases and stakeholders. If the CIP changes an established design then it must outline design issues that motivate a rework. For complex proposals, authors must write a Cardano Problem Statement (CPS) as defined in CIP-9999 and link to it as the `Motivation`. -->
 
 ## Specification
-<!-- The technical specification should describe the proposed improvement in sufficient technical detail. In particular, it should provide enough information that an implementation can be performed solely on the basis of the design in the CIP. This is necessary to facilitate multiple, interoperable implementations. This must include how the CIP should be versioned. If a proposal defines structure of on-chain data it must include a CDDL schema in it's specification.-->
+<!-- The technical specification should describe the proposed improvement in sufficient technical detail. In particular, it should provide enough information that an implementation can be performed solely on the basis of the design in the CIP. This is necessary to facilitate multiple, interoperable implementations. If a proposal defines structure of on-chain data it must include a CDDL schema in its specification.-->
```

Comment:
Think you included changes to this template file by accident in this PR 😎

Reply:
Hmm, I was wondering why I had to do that change twice 😜 ... I guess that's something that can happen when creating 42 PRs in a week 🤪 ... thanks & it's been reverted now. 😎

Reply:
42!! 🤩 youre a super star!